src/OmniSharp.Stdio.Driver/app.manifest

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+
+<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
+  <assemblyIdentity version="1.0.0.0" name="MyApplication.app"/>
```

Comment:
Do we need an `assemblyIdentity` element? If so, should the `version` and `name` match the assembly generated by the OmniSharp.Stdio.Driver project?

Reply:
@JoeRobich The `assemblyIdentity` element is required - https://docs.microsoft.com/en-us/windows/win32/sbscs/application-manifests.

Okay, I'll look into how I get this apart of the build process